templates/request/get/get.handlebars

```diff
@@ -72,3 +72,67 @@
         done();
       });
     });
+{{#if isLoadTest}}
+    it('load tests with {{length description}}', function(done) {
+      arete.loadTest({
+        name: 'PUT THE NAME HERE',
```

Comment:
This is just the report name, I am not sure if we should auto-generate it, if yes, then I can put a normal name like /path load test or something like it